test/teiserver_web/live/account/profile/overview_test.exs

```diff
@@ -75,6 +84,10 @@ defmodule TeiserverWeb.Live.Account.Profile.OverviewTest do
       |> render_click()
 
       assert render(view) =~ "Client is not connected"
+
```

Comment:
very small nitpick: I'd put the assert `lobby_id != nil` right after the call to `make_lobby`.